content/CORE/CORETutorials/Services/ConfiguringOpenVPN.md

```diff
@@ -1,7 +1,10 @@
 ---
 title: "Configuring OpenVPN"
+description: "How to configure OpenVPN"
```

Comment:
Same comment as in preceding articles. This is not a complete sentence and it is too close to the title text.

Reply:
Lines 12-16:
A virtual private network (VPN) is an extension of a private network over public resources. It allows remote clients on a public network to access a private network via a secure connection. TrueNAS provides OpenVPN as a system level service to provide VPN server or client functionality, and can act as a primary VPN server to allow remote clients access to data stored on the system using a single TCP or UDP port. TrueNAS can also integrate into a private network, even when the system is in a separate physical location or only has access to publicly visible networks.

Following level 2 header: Obtaining a Public Key Infrastructure (PKI)

Following level 2 header: Configuring OpenVPN: Process Overview

Following level 3 content already present for Configuring the OpenVPN Client and then OpenVPN Server.

Following level 2 headers for Connection Settings, Security Options and Service Activation.